sphinxcontrib/confluencebuilder/translator.py

```diff
@@ -266,7 +266,9 @@ def visit_enumerated_list(self, node):
         #         'start' attribute of an ordered list tag; limiting to
         #         auto-enumeration items only.
         list_style_type = None
-        if node['enumtype'] == 'upperalpha':
+        if 'enumtype' not in node:
```

Comment:
Can't you use `if 'enumtype' in node and node['enumtype'] == 'upperalpha'` instead? It's more explicit why the first clause is there. I'd find the `pass` confusing without knowing about this issue

Reply:
I have tweaked the change to remove the "is None" check, but opted for saving the condition where if no `enumtype` is provided, the `list_style_type` will not be set to avoid any unnecessary styling modifications. I hope that works for you.